### DISCUSSION TITLE
DL Split - Doge Long Split

### DISCUSSION THREAD

**Discussion by DLSplit** [dogecoin/dogecoin#2984] [General]
I have an idea that I think could really blow Doge Coin out of the park as far increasing market price and being considered more of a viable future Crypto Currency.  

I would rather not just throw my idea out there on a message board though. I would prefer to discuss privately with Doge Company insiders, so that I could get some personal recognition for my input.

I'm open to ideas on how to pursue this endeavor. Feel free to contact me personally.

**Comment by tealethetoolman** [comment]
A true open source warrior. Are you going to send all of your coins to a burn wallet?

**Comment by DLSplit** [comment]
Well it's more of hype philosophy mixed with some programming tech that would/could make Doge a major player in the crypto currency game... hard to say without giving it away. However, I'm looking for a comrade to help me bring this concept to fruition. Give me a call when you get a chance 1-845-728-8322 David



Sent from my Verizon, Samsung Galaxy smartphone



-------- Original message --------
From: tealethetoolman ***@***.***>
Date: 6/11/22 8:16 AM (GMT-05:00)
To: dogecoin/dogecoin ***@***.***>
Cc: DLSplit ***@***.***>, Author ***@***.***>
Subject: Re: [dogecoin/dogecoin] DL Split - Doge Long Split (Discussion #2984)


A true open source warrior. Are you going to send all of your coins to a burn wallet?

**Comment by tealethetoolman** [comment]
This isn’t some proof of stake cryptocurrency with secret backroom dealings. Also, Dogecoin is already a major cryptocurrency.

**Reply by DLSplit** [reply]
Contact me at 845-728-8322 and we can discuss 
David

**Comment by DLSplit** [comment]
There is nothing shady about this at all. But I think it could step Doge up against some of the real power player in the market. I'm considering a web conference to discuss the idea.



Sent from my Verizon, Samsung Galaxy smartphone



-------- Original message --------
From: tealethetoolman ***@***.***>
Date: 6/11/22 1:24 PM (GMT-05:00)
To: dogecoin/dogecoin ***@***.***>
Cc: DLSplit ***@***.***>, Author ***@***.***>
Subject: Re: [dogecoin/dogecoin] DL Split - Doge Long Split (Discussion #2984)


This isn’t some proof of stake cryptocurrency with secret backroom dealings. Also, Dogecoin is already a major cryptocurrency.

**Reply by tealethetoolman** [reply]
Really the best thing that you can do, is fork doge, make your changes, and make a pr explaining your changes so that it could be reviewed and possibly released.

**Reply by DLSplit** [reply]
TealeTheToolMan what do you do for a living if you don't mind me asking?

**Reply by tealethetoolman** [reply]
I eat, drink water/coffee, and move my mouse around from time to time.

**Reply by patricklodder** [reply]
Since we're apparently doing this: same except for the water and I don't have a mouse 😂

**Comment by DLSplit** [comment]
You know how "game theory" is so simple in its nature and large in it's amplification.... well like that sorta thing... hit me up if not you than someone else reading this and let's ponder the possibilities... 845-728-8322 David... no scam... I'd just to work less, drink more, buy more pinball machines, and live careless of money... even donating to the less fortunate...



Sent from my Verizon, Samsung Galaxy smartphone



-------- Original message --------
From: tealethetoolman ***@***.***>
Date: 6/12/22 6:21 PM (GMT-05:00)
To: dogecoin/dogecoin ***@***.***>
Cc: DLSplit ***@***.***>, Author ***@***.***>
Subject: Re: [dogecoin/dogecoin] DL Split - Doge Long Split (Discussion #2984)


Really the best thing that you can do, is fork doge, make your changes, and make a pr explaining your changes so that it could be reviewed and possibly released.

**Reply by DLSplit** [reply]
Maybe you're the guy who could help me fork doge, make changes, and make a pr explaining how it could be reviewed and possibly released... maybe I'm just cog in the wheel with the "blinking eye wiper"
